src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -112,111 +170,259 @@ void run_WithSyncAsArgumentWithAllArg_ShouldExecuteAllSyncers() {
+      try (final SphereClient postTargetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
 
-    final CategoryDraft categoryDraft =
-        CategoryDraftBuilder.of(ofEnglish("foo"), ofEnglish("bar")).key(resourceKey).build();
+        // assertions
+        assertThat(syncerTestLogger.getAllLoggingEvents()).hasSize(2);
 
-    client.execute(CategoryCreateCommand.of(categoryDraft)).toCompletableFuture().join();
+        assertSyncerLoggingEvents(
+            syncerTestLogger,
+            "ProductTypeSync",
+            "Summary: 1 product types were processed in total (1 created, 0 updated "
+                + "and 0 failed to sync).");
 
-    final ProductDraft productDraft =
-        ProductDraftBuilder.of(
-                productType,
-                ofEnglish("foo"),
-                ofEnglish("bar"),
-                ProductVariantDraftBuilder.of().key(resourceKey).sku(resourceKey).build())
-            .key(resourceKey)
-            .build();
+        assertProductTypesAreSyncedCorrectly(postTargetClient);
 
-    client.execute(ProductCreateCommand.of(productDraft)).toCompletableFuture().join();
+        final ZonedDateTime lastSyncTimestamp =
+            assertCurrentCtpTimestampGeneratorAndGetLastModifiedAt(postTargetClient);
 
-    final InventoryEntryDraft inventoryEntryDraft =
-        InventoryEntryDraftBuilder.of(resourceKey, 1L).build();
+        final String sourceProjectKey = postSourceClient.getConfig().getProjectKey();
 
-    client
-        .execute(InventoryEntryCreateCommand.of(inventoryEntryDraft))
-        .toCompletableFuture()
-        .join();
+        assertLastSyncCustomObjectIsCorrect(
+            postTargetClient,
+            sourceProjectKey,
+            "productTypeSync",
+            ProductSyncStatistics.class,
+            lastSyncTimestamp);
+
+        cleanUpProjects(postSourceClient, postTargetClient);
+      }
+    }
+  }
+
+  private static void assertProductTypesAreSyncedCorrectly(@Nonnull final SphereClient ctpClient) {
+
+    final PagedQueryResult<ProductType> productTypeQueryResult =
+        ctpClient.execute(ProductTypeQuery.of().byKey(RESOURCE_KEY)).toCompletableFuture().join();
+
+    assertThat(productTypeQueryResult.getResults())
+        .hasSize(1)
+        .hasOnlyOneElementSatisfying(
+            productType -> assertThat(productType.getKey()).isEqualTo(RESOURCE_KEY));
+  }
+
+  private void assertLastSyncCustomObjectIsCorrect(
+      @Nonnull final SphereClient targetClient,
+      @Nonnull final String sourceProjectKey,
+      @Nonnull final String syncModule,
+      @Nonnull final Class<? extends BaseSyncStatistics> statisticsClass,
+      @Nonnull final ZonedDateTime lastSyncTimestamp) {
+
+    final CustomObjectQuery<LastSyncCustomObject> lastSyncQuery =
+        CustomObjectQuery.of(LastSyncCustomObject.class)
+            .byContainer(format("commercetools-project-sync.%s", syncModule));
```

Comment:
I would use APPLICATION_DEFAULT_NAME for this one. also "commercetools-project-sync.%s" is duplicated 3 times in same file

Reply:
4d00341